manifests/w/Wazuh/WazuhAgent/4.2.5/Wazuh.WazuhAgent.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Wazuh.WazuhAgent
+PackageVersion: 4.2.5
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
+  InstallerUrl: https://packages.wazuh.com/4.x/windows/wazuh-agent-4.2.5-1.msi
+  InstallerSha256: 3C42ACC47D7B0DF07D25884BB5093FA64B0625B4BB9D2D8198AEB4570B4903B6
+  ProductCode: '{DC32A5AC-7AEC-4A24-AB7B-3C789CD379E2}'
```

```suggestion
  ProductCode: '{B874D271-7B8F-4F6A-8075-0265B962B8CD}'
```
wingetcreate or YamlCreate.ps1 should automatically update this for you.